### PR TITLE
Restore work language list

### LINKED ISSUE #175: Empty languages list when creating a new work
"Relate checked recordings to" function can create new works with a specified type and language. However, the languages selection list is empty. There is no way to specify needed language.

### PR DESCRIPTION
**Pull request by jesus2099**
Load **language list** from `/release/add` page, as it is no longer available from `/work/create` page.

This would fix #175.

### CODE REVIEW

**Review by kellnerd — CHANGES_REQUESTED** [review]
I think I had once tried to adapt this code to use the new work language select input but failed miserably. Clever idea to simply use the select input from the release page!
The only caveat which I can see with this approach is that [no lyrics] is missing. You could manually inject this one entry if you want, but it is also very useful as is already.

More important is the work submission code which also has to be adapted since the field name has changed from `language_id` to `languages.#` as multiple lyric languages are supported now:

[These lines](https://github.com/murdos/musicbrainz-userscripts/pull/536/files#diff-7778dd3ca361371e4b03ab85e352df89b6d713c16a2fa184d7cb5b7ca3045d95L1102-L1104) have to be replaced with the following (successfully tested):

```js
                    if (kind === 'type') {
                        data += `&edit-work.${kind}_id=${$obj.val()}`;
                    } else if (kind === 'language') {
                        data += `&edit-work.languages.0=${$obj.val()}`;
                    }
```

**Comment by jesus2099** [comment]
> the field name has changed from `language_id` to `languages.#`

Yes I saw that when testing the script.
I fixed and force pushed, maybe at the same time as your remark.

Thanks for all your helps.
I will read them all now and check them later. 😁👍

File: batch-add-recording-relationships.user.js [hunk 1]
```diff
@@ -384,16 +384,26 @@ function batch_recording_rels() {
...
+            .on('change', function () {
+                setting(`work_type`, this.value);
+            });
+    });
+
+    // Get actual (release) languages
+    ws_requests.unshift_get('/release/add', function (data) {
+        let nodes = $.parseHTML(data);
+        $work_options.language
+            .append($(`select#language option[value]`, nodes))
```
[the remark this answers is not in the document]
Comment (jesus2099) [reply batch-add-recording-relationships.user.js:402]: No reasons, I just didn't know how that library worked and tried some random things. 🤣

**Comment by jesus2099** [comment]
Currently [being tested](https://community.metabrainz.org/t/edit-notes-for-batch-add-performance-of-relationships/642993/27?u=jesus2099)  and several fixes are planned.

**Comment by jesus2099** [comment]
> Clever idea to simply use the select input from the release page!

I just mimicked how it was done, I would have not found it alone, maybe. :)

But now I will call `/work/create?edit-work.languages.0=486` to get the full work language list, including `[No lyrics]`, **properly localised**!

File: batch-add-recording-relationships.user.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jesus2099) [reply batch-add-recording-relationships.user.js]: I have added this as part of my second commit (13ff88e3356e1976c780451a4703a6be153ec3f2).
I hope I am not breaking anything with that change request system.

**Comment by jesus2099** [comment]
I have changed all requested stuff, @kellnerd. :)

File: batch-add-recording-relationships.user.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kellnerd) [reply batch-add-recording-relationships.user.js]: Excellent, I hadn't realized that `zxx` was already there, just buried in the lower section of the list and under a different description :+1:

**Review by kellnerd — APPROVED** [review]
I think this PR is ready to be released

*Edit*: I'll wait with merging until #535 is ready to avoid a potential version conflict.

**Comment by jesus2099** [comment]
For info, instead of work language list's `[No lyrics]` we have the release language list's `[No linguistic content]` which has the same key, anyway.

I prefer that that hard-coding `[No lyrics]` in the code, that would slightly _break_ language consistency for non-English MB editors. :)

**Comment by jesus2099** [comment]
I guess it's easier to merge this one. And then to merge #535.
Isn't it?

**Comment by kellnerd** [comment]
Sure, I was just wondering about the version number, of which I don't really understand the fourth component 2023.6.30.**19** and which version the other PR will get.
But maybe I should simply merge this PR now and you can assign the "final" version of #535 a ".1" as the fourth number again like 2023.6.31.1 (fake date) or 2023.7.1.1 since that will probably the last release for a few months/years again.

**Comment by jesus2099** [comment]
19 was for 00:19.
I think currently, timestamp for the other PR is more recent, so I think it's easier to merge here first, then the other. 😉

**Pull request merged**